packages/cli/bin/index.js

```diff
@@ -10,6 +10,11 @@ const start = require('../lib/start');
 const devPlugin = require('../lib/devPlugin');
 const dataset = require('../lib/dataset');
 
+const path = require('path');
+
+process.env.NODE_PATH = path.join(process.cwd(), 'node_modules');
+require("module").Module._initPaths();
```

Comment:
Why do we change the NODE_PATH?

Reply:
to change the order node searches node_modules so that node_modules in current working directory would be searched. (because pipcook-cli is installed globally but plugins are installed locally)

Reply:
The working directory `node_modules` should be in the search paths by default as:

```
NODE_MODULES_PATHS(START)
1. let PARTS = path split(START)
2. let I = count of PARTS - 1
3. let DIRS = [GLOBAL_FOLDERS]
4. while I >= 0,
   a. if PARTS[I] = "node_modules" CONTINUE
   b. DIR = path join(PARTS[0 .. I] + "node_modules")
   c. DIRS = DIRS + DIR
   d. let I = I - 1
5. return DIRS
```